ListManagement/processOpenSlidesParticipants.py

```diff
@@ -0,0 +1,184 @@
+    class OpenSlidesCols:
+        TITLE = "title"
+        FIRST_NAME = "first_name"
+        LAST_NAME = "last_name"
+        EMAIL = "email"
+        PRONOUN = "pronoun"
+        GENDER = "gender"
+        USERNAME = "username"
+        DEFAULT_PASSWORD = "default_password"
+        IS_ACTIVE = "is_active"
+        IS_PHYSICAL_PERSON = "is_physical_person"
+        SAML_ID = "saml_id"
+        IS_PRESENT = "is_present"
+        STRUCTRE_LEVEL = "structure_level"
+        NUMBER = "number"
+        VOTE_WEIGTH = "vote_weight"
+        GROUPS = "groups"
+        COMMENT = "comment"
+        TOP_ROW = [TITLE, FIRST_NAME,LAST_NAME,EMAIL,PRONOUN,GENDER, USERNAME, DEFAULT_PASSWORD, IS_ACTIVE, IS_PHYSICAL_PERSON,SAML_ID,IS_PRESENT,STRUCTRE_LEVEL,NUMBER,VOTE_WEIGTH,GROUPS,COMMENT]
+    class ANCols:
+        FIRST_NAME = "first_name"
+        LAST_NAME = "last_name"
+        EMAIL = "email"
+        PRONOUNS = "pronouns"
+
+class CommmandFlags:
+    PREVIOUS_LIST = "-p"
+    PREVIOUS_LIST_LONG = "--prev-list"
+    MEMBERSHIP_LIST = "-m"
+    MEMBERSHIP_LIST_LONG = "--membership-list"
+    OUTPUT = "-o"
+    OUTPUT_LONG = "--output"
+    HELP = "-h"
+    HELP_LONG = "--help"
+    USAGE = "usage: python (python3 on mac) processOpenSlidesParticipants.py -p exportFromLastGBM.csv -m currentMembershipListFromActionNetwork.csv -o output-csv"
+
+    def __init__(self, prevListPath, membershipListPath, outputPath) -> None:
+        self.prevListPath = prevListPath
+        self.mebershipListPath = membershipListPath
+        self.outputPath = outputPath
+
+def setup():
+    if not os.path.exists(Constants.WORKING_DIR):
+        os.mkdir(Constants.WORKING_DIR)
+    logging.basicConfig(filename=Constants.LOG_PATH, level=logging.INFO, format="%(asctime)s : %(levelname)s : %(message)s")
+    logging.getLogger().addHandler(logging.StreamHandler(sys.stdout))
+    logging.info("Setup Complete")
+
+def parseArgs():
+    parser = argparse.ArgumentParser(description=CommmandFlags.USAGE)
+    parser.add_argument(CommmandFlags.PREVIOUS_LIST, CommmandFlags.PREVIOUS_LIST_LONG, required=True)
+    parser.add_argument(CommmandFlags.MEMBERSHIP_LIST, CommmandFlags.MEMBERSHIP_LIST_LONG, required=True)
+    parser.add_argument(CommmandFlags.OUTPUT, CommmandFlags.OUTPUT_LONG, required=True)
+    parsedArgs = parser.parse_args()
+    return CommmandFlags(prevListPath=parsedArgs.prev_list, membershipListPath=parsedArgs.membership_list, outputPath=parsedArgs.output)
+
+@dataclasses.dataclass
+class OpenSlidesRow:
+    firstName: str
+    lastName: str
+    email: str
+    title: str = ""
+    pronoun: str = ""
+    gender: str = ""
+    username: str = ""
+    defaultPassword: str = ""
+    isActive: int = 1
+    isPhysicalPerson: int = 1
+    saml_id: str = ""
+    isPresent: str  = ""
+    structreLevel : str = ""
+    number : str = ""
+    voteWeight : int = 1
+    groups: str = "Delegates"
+    comment: str = ""
+
+    def toRow(self):
```

Comment:
I think data classes have a built-in methods `asTuple ` and `asList`. Or maybe this would be simpler as `[value for value in self.__dict__.values()]`